src/content/cloud/personal-access-tokens.mdx

```diff
@@ -11,6 +11,8 @@ You can use Personal Access Tokens instead of OAuth to authenticate with Okteto.
 
 Personal Access Tokens are useful when scripting, or when configuring [Preview Environments](cloud/preview-environments/overview.mdx).
 
+> Tokens are valid for 180 days since their creation. 
```

Comment:
"A token is valid for 180 days. Once it expires, you'll need to delete it and create a new one."